src/Dapr.Configuration.Extensions/DaprSecretDescriptor.cs

```diff
@@ -0,0 +1,46 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.DaprSecretStore;
+
+namespace Microsoft.Extensions.Configuration
```

Comment:
I'm a little iffy about this being in the M.E.Configuration namespace since it's not an extension, and it's not really useful outside of the context of dapr. 

It might be worthwhile to move this to the `Dapr` namespace and then have an API like `AddDaprSecretStore(string storeName, string[] secrets)` as the simplest alternative.

Reply:
`AddDaprSecretStore(string storeName, string[] secrets)` will only work for stores that do not require metadata such as Azure Key Vault. 

I thought of using something like `Dictionary<string, Dictionary<string,string>>` for the secrets parameter but did not seem like a clean approach.

@rynowak thoughts?

Reply:
I've removed `DaprSecretDescriptor` in this feauture branch: https://github.com/cmendible/dotnet-sdk/blob/0402ac4b621e04dd98a6131e2723fde7642a69dc/src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationExtensions.cs#L28 in favor of and `IDaprSecretStoreManager` implementation inspired in the old [AzureKeyVaultConfigurationProvider](https://github.com/aspnet/Configuration/blob/f64994e0655659faefccead7ccb5c1edbfd4d4ba/src/Config.AzureKeyVault/AzureKeyVaultConfigurationProvider.cs#L26)

@rynowak, @davidfowl this approach is more flexible, and cleaner for those using secret stores which don't require metadata (i.e Azure Key Vault). Thoughts?

Reply:
Solved changing the namespace to Dapr.Extensions.Configuration in 2bef7db